build/version/version.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package version provides information about FerretDB version and build configuration.
```

Comment:
Why `build` was moved out of `internal`? Is it just for consistency and to have all the build-related things and version text files (version, commit, branch) in the same directory?  Or are there other reasons too?

Reply:
The whole point of this issue/PR to provide a stable public interface for builds and version information, and "internal" is not public